src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java

```diff
@@ -272,8 +272,10 @@ private void errorHandler(String error, EmbedBuilder embedBuilder, MessageReceiv
                     for (StackLine line : exception.getLines()) {
                         if (line.getStackPackage() != null && line.getStackPackage().startsWith("org.geysermc") && !line.getStackPackage().contains("shaded")) {
                             // Get the file url
+                            if(line.getLine()!=null){
                             String lineUrl = fileFinder.getFileUrl(line.getSource(), Integer.parseInt(line.getLine()));
-
+                                continue;
```

Comment:
Why the continue?

Reply:
The null value of getLine function would give us a faulty value for the lineUrl right.I have provided the continue so that the execution in the case of the faulty behaviour is skipped.
If it is not required I can remove that.

Reply:
I don't think this PR will even compile - the code below this, outside your if statement, uses `lineUrl`

You could simply add `line.getLine() != null` as another loop condition on line 273 without these changes

Reply:
I have made the changes on line 273 and removed my last changes. @Konicai